extensions/notification/utils/auth.js

```diff
@@ -64,6 +64,10 @@ function generateKafkaAuthObject(auth) {
         const keytabPath = getAuthFilePath(keytab);
         if (keytabPath) {
             authObject['sasl.kerberos.keytab'] = keytabPath;
+            //optional parameter, default kinit command
+            const kinitCommand = `kinit -R -k ${principal} -t ${keytabPath} ||
```

Comment:
just double checking, is this command always run in a shell environment? in order to interpret the double command and `||`.

Reply:
Yes, https://github.com/edenhill/librdkafka/blob/v1.5.3/CONFIGURATION.md

Reply:
Had to check it, I think using just `kinit -k -t...` should be fine. Changing it.